app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
@gberginc I agree with your hierarchy. @gberginc @agrare My question was only to start thinking about how user should understand it. Must the terminology be same or more similar as in vCD (because users will migrate from VMware cloud to MiQ) or more similar to Amazon/Openstack terminology (users will move from other providers to VMware). In anyway it could happen that the terminology won't be well accepted by the users.